packages/components/src/toggle-control/index.js

```diff
@@ -40,7 +40,7 @@ class ToggleControl extends Component {
 
 		return (
 			<BaseControl
-				label={ label }
+				label={ heading }
```

Comment:
I'm not sure I understand why we need the changes in this component? cc @jasmussen ?

Reply:
If my assumption is correct you wanted to align the label at the right so you duplicated it and passed something inexisting to the parent component? I tweaked the component and use `order` CSS instead.

Reply:
👍👍